src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="header">
+        <a href="#">
```

Comment:
The link should have a class `is-active` for the first navigation link (`Apple`) as per the task requirements.